### ISSUE TITLE
Slicing is not working Properly

### ISSUE THREAD

**Issue by thaboub** [epit3d/spycer#15]
Hi Dear
I am really interested to this slicer software, but it is not fully functioning, and I am not sure what it is missing. 
I am using Windows 10, I built all “goosli” binaries, and put them as recommended with 'data' directory to spycer as it is shown in the below picture. "main.py" runs fine, but when I upload the STL file and try to slice it, it generates the error showns below 
Can you please help? 

Thanks

![image](https://user-images.githubusercontent.com/32969483/117050589-60108a00-acca-11eb-8952-d22978e77f46.png)

![image](https://user-images.githubusercontent.com/32969483/117050650-71599680-acca-11eb-93a0-29cef81b712d.png)

**Comment by l1va** [comment]
Hi Abdel, 
seems your Anaconda do not have permissions to call outer process ( goosli ). You need to configure rights.

**Comment by thaboub** [comment]
Hi Mike,
the slicing is still not working yet although I tried this time the installation in Ubuntu instead of Windows 10. I followed all recommended steps by installing spycer and building “goosli” binaries in Ubuntu. The spycer runs fine in uploading and moving the STL file as well as adding and moving the planes through the uploaded STL file. However, the slicing “Standard 3a slicing” and “vip slicing” are not functioning as they generated the errors shown below. Any workaround, hints, or help in solving this issue is highly appreciated.
Thanks a [
![errors](https://user-images.githubusercontent.com/32969483/120764807-65c9dd00-c4cd-11eb-90fd-8b29721a3649.jpg)
](url)

![files](https://user-images.githubusercontent.com/32969483/120764848-71b59f00-c4cd-11eb-9216-962db035f84f.jpg)
